src/lib.rs

```diff
@@ -58,6 +58,11 @@ where
         Ok(Cc1101(lowlevel::Cc1101::new(spi)?))
     }
 
+    /// Last Chip Status Byte
+    pub fn get_last_chip_status_byte(&mut self) -> Option<StatusByte> {
```

Comment:
Maybe the name could be trimmed to `get_chip_status` as `get_last_chip_status_byte` kind of duplicates the return type without adding any meaning, and that it's the last is kind of assumed, and Option adds the meaning that it's maybe not available.

Reply:
We can try to settle at `get_chip_status` and hope that the return type and the doc comment clarify the full meaning of the function